lib/guardian/plug.ex

```diff
@@ -117,6 +117,7 @@ defmodule Guardian.Plug do
     case Guardian.encode_and_sign(object, type, new_claims) do
       {:ok, jwt, full_claims} ->
         conn
+        |> Plug.Conn.configure_session(renew: true)
```

Comment:
I was wondering about this myself but I didn't see anything in the docs, I can swap it up.